corehq/apps/data_cleaning/models.py

```diff
@@ -384,29 +384,15 @@ def edited_value(self, case, edited_properties=None):
         properly layer all changes in order.
         """
         edited_properties = edited_properties or {}
-        regex_transformations = {
-            EditActionType.FIND_REPLACE: lambda x: re.sub(
-                re.compile(self.find_string), self.replace_string, x
-            ),
-        }
-
-        if self.use_regex and self.action_type in regex_transformations:
-            old_value = edited_properties.get(self.prop_id, case.get_case_property(self.prop_id))
-            return regex_transformations[self.action_type](old_value)
+        old_value = edited_properties.get(self.prop_id, case.get_case_property(self.prop_id))
 
         simple_transformations = {
```

Comment:
Naming nitpick: these could be called "overwrites," that's how I started thinking about them while reading through the tests.